Algorithm/Examples/LiveFeaturesAlgorithm.cs

```diff
@@ -44,7 +45,7 @@ public override void Initialize()
             AddData<Bitcoin>("BTC", Resolution.Second);
         }
 
-        //New Bitcoin Data Event:
+        /// <inheritdoc />
```

Comment:
Where possible please limit use of <inheritdoc>, we must assume its a human reading the file and provide good documentation on each method as much as possible for new people. I know its ugly copying-pasting; but its the lesser of two evils -- make code unreadable for humans, or copy-paste a few documentation snippets? Copying and pasting costs a little more work but will make life easier for the millions of people who follow after us,